src/anemoi/training/train/forecaster.py

```diff
@@ -127,6 +136,53 @@ def __init__(
     def forward(self, x: torch.Tensor) -> torch.Tensor:
         return self.model(x, self.model_comm_group)
 
+    @staticmethod
+    def get_loss_function(
+        config: DictConfig,
+        **kwargs,
+    ) -> Union[torch.nn.Module, torch.nn.ModuleList]:  # noqa: FA100 # stop ruff
+        """
+        Get loss functions from config.
+
+        Will include additional kwargs if specified in the config.
+        Can be ModuleList if multiple losses are specified.
+
+        E.g.
+            If `include_node_weights: True` is set in the config, and node_weights in kwargs
+             `node_weights` will be included in the config to instantiate the module with.
+        """
+        config_container = OmegaConf.to_container(config, resolve=False)
+        if isinstance(config_container, list):
+            return torch.nn.ModuleList(
+                [
+                    GraphForecaster.get_loss_function(
+                        OmegaConf.create(loss_config),
+                        **kwargs,
+                    )
+                    for loss_config in config
+                ],
+            )
+
+        loss_config = OmegaConf.to_container(config, resolve=True)
+
+        # Create loss_config including elements from kwargs if they
+        # are specified in the config with include_{key}: True
+        loss_init_config = {}
+        for key in loss_config:  # Go through all keys given in the config
+            # If key does not start with include_, add it to the loss_init_config
+            if not key.startswith("include_"):
```

Comment:
What is the reason to do this indirectly and not just use the argument directly in the init?